patches/tModLoader/Terraria/GameContent/UI/Elements/UIBestiaryEntryGrid.cs.patch

```diff
@@ -0,0 +1,20 @@
+--- src/Terraria/Terraria/GameContent/UI/Elements/UIBestiaryEntryGrid.cs
++++ src/tModLoader/Terraria/GameContent/UI/Elements/UIBestiaryEntryGrid.cs
+@@ -47,7 +_,7 @@
+ 			for (int j = 0; j < maxEntriesHeight; j++) {
+ 				for (int k = 0; k < maxEntriesWidth; k++) {
+ 					if (num2 >= list.Count)
+-						break;
++						return;
```

Comment:
Curious as to why this has to change

Reply:
Redundant patch, please remove. Looks like you previously had something at the bottom of this method but since reworked it